test/epochtalk_server_web/controllers/thread_test.exs

```diff
@@ -21,23 +23,30 @@ defmodule Test.EpochtalkServerWeb.Controllers.Thread do
         [category: category, view_order: 0],
         [board: board, category: category, view_order: 1],
         [board: admin_board, category: category, view_order: 2],
-        [board: super_admin_board, category: category, view_order: 3]
+        [board: super_admin_board, category: category, view_order: 3],
+        [board: test_move_board, category: category, view_order: 4]
       ]
     )
 
     factory_threads = build_list(3, :thread, board: board, user: user)
 
+    num_thread_replies = 3
+    thread_post_count = num_thread_replies + 1
     thread = build(:thread, board: board, user: user)
+    _thread_posts = build_list(num_thread_replies, :post, user: user, thread: thread.post.thread)
```

Comment:
would be good to validate return values using this variable - otherwise it can be removed, and use a comment instead